spswarehouse/powerschool/powerschool.py

```diff
@@ -0,0 +1,437 @@
+import logging
+import time
+import os
+
+from ducttape.utils import (
```

Comment:
ducttape is not currently a requirement of spswarehouse.

We can make it a requirement easily enough, but wondering if its worthwhile for a single function.

Reply:
Adapting to remove this dependency. Will resolve this conversation when I've tested it works and committed the change.